v1/format/structured_cloud_event.h

```diff
@@ -0,0 +1,24 @@
+#ifndef STRUCTURED_CLOUD_EVENT_H
+#define STRUCTURED_CLOUD_EVENT_H
+
+#include "proto/cloud_event.pb.h"
+
+namespace cloud_events {
+namespace format {
+
+enum CloudEventFormat {JSON};
```

Comment:
use enum classes: https://abseil.io/tips/86

consider moving it inside the class if not used anywhere else

Reply:
https://google.github.io/styleguide/cppguide.html#Declaration_Order